CONTRIBUTING.md

```diff
@@ -20,16 +20,16 @@ Before reading: All of the rules below are guidelines, which means that they sho
 
 #### Issues:
  - Please use the BUG/ISSUE template
- - Please be clear when describing issues.
+ - Please be sure to be clear when describing issues.
```

Comment:
unnecessary, cumbersome.

maybe try: "Please be as clear as possible when ..."